package.json

```diff
@@ -176,7 +175,7 @@
     "string-width": "^2.1.1",
     "strip-ansi-stream": "^1.0.0",
     "tar-stream": "^1.6.2",
-    "tar-utils": "^1.1.0",
+    "tar-utils": "^2.0.0",
```

Comment:
All dependencies (`resin-bundle-resolve`, `resin-compose-parse`, `tar-utils`, `resin-multibuild`) are in place, including changes to `resin-builder` that have been deployed to staging earlier today.